docs/excel/excel-add-ins-ranges-advanced.md

```diff
@@ -264,6 +264,48 @@ Excel.run(async (context) => {
 
 ![Data in Excel after range’s remove duplicates method has been run](../images/excel-ranges-remove-duplicates-after.png)
 
+## Group data for an outline
+
+> [!NOTE]
+> The outline APIs for grouping rows and columns are currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+
+Rows or columns of a range can be grouped together to create an [outline](https://support.office.com/article/Outline-group-data-in-a-worksheet-08CE98C4-0063-4D42-8AC7-8278C49E9AFF). These groups can be collapsed and expanded to hide and show the corresponding cells. This makes quick analysis of top-line data easier. Use [Range.group](/javascript/api/excel/excel.range#group-groupoption-) to make these outline groups.
+
+An outline has a hierarchy. Smaller groups nest under larger groups. This allows the outline to be viewed at different levels. Changing the visible outline level can be done programmatically through the [Range.showOutlineLevels](/javascript/api/excel/excel.range#showOutlineLevels-rowLevels--columnLevels-) method. Note that Excel only supports eight levels of outline groups.
```

Comment:
The target of the link `[Range.showOutlineLevels](/javascript/api/excel/excel.range#showOutlineLevels-rowLevels--columnLevels-)` doesn't seem to exist yet -- assuming it just hasn't made its way into the ref docs yet, but will soon?

Reply:
No worries! I knew you were probably accounting for this already :)